parser/parser.y

```diff
@@ -278,6 +278,7 @@ import (
 	value		"VALUE"
 	values		"VALUES"
 	variables	"VARIABLES"
+	version		"VERSION"
```

Comment:
"version" is not a MySQL reserved  keyword. So it can be used as an identifier. 
You need to put it into NotKeywordToken rule: https://github.com/pingcap/tidb/blob/master/parser/parser.y#L1691
Then add a test case here: https://github.com/pingcap/tidb/blob/master/parser/parser_test.go#L44